### PR TITLE
feat: post report as comment

### PR DESCRIPTION
**Pull request by dkrutskikh**

### CODE REVIEW

**Comment by dkrutskikh** [comment]
test

**Comment by dkrutskikh** [comment]
test comment

**Review by incendial — CHANGES_REQUESTED** [review]

File: action.yml [hunk 1]
```diff
@@ -34,6 +34,10 @@ inputs:
     description: "Configurable unused files report title pattern."
     required: false
     default: "Dart Code Metrics unused files report of $packageName"
+  report_in_review_comment:
+    description: "Publish reports results to GitHub PullRequest review comments."
```
[the remark this answers is not in the document]
Comment (dkrutskikh) [reply action.yml:38]: fixed

File: action_app/lib/src/task.dart [hunk 2]
```diff
@@ -170,7 +188,78 @@ class GitHubTask
...
+          .first
+          .replaceAll(_onlySymbolsRegex, '')
+          .toLowerCase();
+
+      return commentHeader == currentHeader;
+    });
+  }
+
+/*
+export async function getPreviousComment(
```
[the remark this answers is not in the document]
Comment (dkrutskikh) [reply action_app/lib/src/task.dart:246]: removed

**Review by incendial — APPROVED** [review]

**Pull request merged**